test/run-tests.sh

```diff
@@ -20,6 +20,7 @@ trap cleanup EXIT
 rm -Rf /tmp/s3proxy
 mkdir -p /tmp/s3proxy
 
+java -version
```

Comment:
Needed?

Reply:
It is not so important. However, java in transition now. I added it since I wanted to confirm test behavior and java version on travis-ci worker when I would update travis.yaml.